File: tests/eval_c/362.c
#include <assert.h>
#include <string.h>

void f362(const char *s) {
    int i = 0;
    assert(s[i] == 'a');
    while (i < strlen(s) - 1) {
        i += 1;
    }
    assert(s[i] == 'b');
}

File: benchmarks/oopsla25/subjects/362.py
import sys

def f362(s: str):
  i = 0  
  assert s[i] == "a"
  while i < len(s)-1:
    i += 1
  assert s[i] == "b"

if __name__ == '__main__':
  my_string = ""
  if len(sys.argv) == 1:
    my_string = sys.stdin.read()
  else:
    fd = open(sys.argv[1], 'r')
    my_string = fd.read()
    fd.close()
  
  f362(my_string)
